uhabits-core/src/jvmMain/java/org/isoron/uhabits/core/models/Entry.kt

```diff
@@ -27,7 +27,7 @@ data class Entry(
         /**
          * Value indicating that the habit is not applicable for this timestamp.
          */
-        const val SKIP = 3
+        const val SKIP = -2
```

Comment:
Why this change?

I think it's because of the other change where you want to match a whole range (`>0`), but I think it's not a good idea. This object is basically an enum, I think it's better to match every single case, so that it will be easier to see what needs to change if we add a new possible value.

Reply:
But if I leave it as it is, if user inputs 0.003 as an entry, it might mistake it for a skipped day

Reply:
Oh wait a second we use only 2 decimal places for the entries - you're right, I'll correct it later

Reply:
This change by itself (without a corresponding database migration) would also break all existing Loop databases. Since `0.003` cannot be entered by the user, I think we can use it for now. In the future, we should probably split this into two fields. For example, if the user enters a value, then `field1=YES_MANUAL`, `field2=500.0`. If the user skips, then `field1=SKIP`, `field2=<ignored>`.